.github/workflows/build-docs.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - uses: aica-technology/.github/.github/actions/ghcr-ensure-prefix@v0.6.1
         id: ensure-image
         with:
-          image_name: aica-technology/backend-api-docs
+          image_name: aica-technology/api-server-docs
```

Comment:
Just made a change here so I can comment but what is going on here? This backend-api-docs is deprecated and the api version file in this repo contains 3.0.0 which is behind too no?

Reply:
yes you are right that these are both outdated. `api-server-docs` is the newer image and v4.3.2 is the latest. For now l would say to use `api-server-docs:latest` as the image of choice, but that we add a new issue to take care of this properly:

I think we should:
- always pull `latest` for the default API reference link
- also pull multiple older tags of the image and put them in respectively tags subdirectories so that we can include version-specific links in a dropdown
  - I think we can just manually set the latest of each prior major release, so e.g. 3.4.1, then 4.X before v5, etc, while latest is always latest
- also bundle our openAPI schema in the image so that we distribute it more easily